definitions/npm/history_v4.x.x/test_history_v4.x.x.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import createBrowserHistory from 'history/createBrowserHistory';
+import createMemoryHistory from 'history/createMemoryHistory';
+import createHashHistory from 'history/createHashHistory';
+
+// browser history
+const browser = createBrowserHistory({
+  basename: "",
+  forceRefresh: false,
+  keyLength: 6,
+})
+
+browser.location.pathname
```

Comment:
Provide expected typings on all these checks. for better and more explicit tests.

Reply:
done